packages/node_modules/@webex/plugin-logger/src/logger.js

```diff
@@ -40,12 +40,20 @@ const authTokenKeyPattern = /[Aa]uthorization/;
 /**
  * Recursively strips "authorization" fields from the specified object
  * @param {Object} object
+ * @param {Array<mixed>} visited
  * @private
  * @returns {Object}
  */
-function walkAndFilter(object) {
+function walkAndFilter(object, visited = []) {
+  if (visited.includes(object)) {
+    // Prevent circular references
+    return '[Circular]';
```

Comment:
Originally used `undefined` but switched over to `'[Circular]'` as it seems more informative